### PR TITLE
Fix the ValueCreator API change

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2864

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.2.3-SNAPSHOT
+version=1.3.0-SNAPSHOT
 #dependency versions
-ballerinaLangVersion=2201.0.1
+ballerinaLangVersion=2201.1.0-rc1.1
```
Comment (niveathika) [inline gradle.properties:5]:
```suggestion
ballerinaLangVersion=2201.1.0-rc1.2
```

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "grpc"
-version = "1.2.3"
+version = "1.3.0"
 distribution = "2201.0.1"
```
Comment (niveathika) [inline ballerina/Ballerina.toml:5]:
```suggestion
distribution = "2201.1.0"
```

Comment (BuddhiWathsala) [reply ballerina/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/797/commits/fc6df8c0cc28db1bfd3e5c8bbbba82fd41ee5f9c

File: gradle.properties (hunk 1 above)
Comment (BuddhiWathsala) [reply gradle.properties]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/797/commits/d392fe81bc4e919e2964dc387581557991582525

**Review by niveathika — APPROVED** [review]

**Pull request merged**